client.go

```diff
@@ -53,24 +53,6 @@ const (
 // it so River can check that inserted job kinds have a worker that can run
 // them.
 type Config struct {
-	// AdvisoryLockPrefix is a configurable 32-bit prefix that River will use
-	// when generating any key to acquire a Postgres advisory lock. All advisory
-	// locks share the same 64-bit number space, so this allows a calling
-	// application to guarantee that a River advisory lock will never conflict
-	// with one of its own by cordoning each type to its own prefix.
-	//
-	// If this value isn't set, River defaults to generating key hashes across
-	// the entire 64-bit advisory lock number space, which is large enough that
-	// conflicts are exceedingly unlikely. If callers don't strictly need this
-	// option then it's recommended to leave it unset because the prefix leaves
-	// only 32 bits of number space for advisory lock hashes, so it makes
-	// internally conflicting River-generated keys more likely.
-	//
-	// Advisory locks are currently only used for the fallback/slow path of
-	// unique job insertion where finalized states are included in a ByState
-	// configuration.
-	AdvisoryLockPrefix int32
```

Comment:
Once we complete the deprecation and eventual removal of the advisory lock based unique jobs, would you want to just nuke this config altogether? It's possible we could want to use advisory locks again at some point but it'd be easy enough to re-add.